src/Domain/CoordinateMaps/BulgedCube.hpp

```diff
@@ -431,15 +434,20 @@ namespace CoordinateMaps {
  * \end{bmatrix}
  * \f]
  *
- * In the case where the same coordinates are used for the cube and the sphere,
- * we have \f$\xi = \Xi\f$, etc. In this case, the formula reduces further.
- * It is given by:
+ * In the above formula, the lower case Greek coordinates are artifacts of
+ * using linear (as opposed to nonlinear) interpolation in the direction
+ * between the two bounding surfaces, while the upper case Greek coordinates
+ * are the coordinates along these surfaces which can be specified to be either
+ * equidistant(linear) or equiangular(nonlinear). In the case where the
+ * (non)linearity of the interpolation is chosen to match that of the
+ * coordinates along the surface, we have \f$\xi = \Xi\f$, etc. In this case,
+ * the formula reduces further.It is given by:
  *
```

Comment:
Is this final formula below the one that is always used in the code?  If so, please say so.